panels/class-debug-bar-panel.php

```diff
@@ -60,4 +60,38 @@ function title( $title = null ) {
 	function debug_bar_classes( $classes ) {
 		return $classes;
 	}
+
+	/**
+	 * Render a list of logged error notices.
+	 *
+	 * Used by the Debug_Bar_PHP panel and the Debug_Bar_Deprecated panel to
+	 * display lists of PHP Warnings, PHP Notices and WP Deprecated calls.
+	 *
+	 * @since 0.10.0
+	 *
+	 * @param array  $errors      Array of recorded errors/notices/calls.
+	 * @param string $line_prefix Text string to prefix the recorded message with.
+	 * @param string $class       CSS class suffix for the list item.
+	 */
+	protected function render_error_list( $errors, $line_prefix, $class ) {
+		if ( count( $errors ) ) {
+			echo '
+			<ol class="debug-bar-error-notices-list">';
+
+			foreach ( $errors as $location_message_stack ) {
+				list( $location, $message, $stack ) = $location_message_stack;
+
+				echo '
+				<li class="debug-bar-', esc_attr( $class ), '">
+					<strong>', esc_html( $line_prefix ), '</strong> ',
```

Comment:
I understand you prefer commas to string things together, but could we maintain WordPress' way here with space-padded periods?

```<strong>' . esc_html( $line_prefix ) . '</strong> '```